SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -1371,6 +1371,8 @@ void CFlowOutput::AddAerodynamicCoefficients(const CConfig* config) {
   AddHistoryOutput("REFERENCE_FORCE", "RefForce", ScreenOutputFormat::FIXED, "AERO_COEFF", "Reference force used to compute aerodynamic coefficients", HistoryFieldType::COEFFICIENT);
   /// DESCRIPTION: Drag coefficient
   AddHistoryOutput("DRAG",       "CD",   ScreenOutputFormat::FIXED, "AERO_COEFF", "Total drag coefficient on all surfaces set with MARKER_MONITORING", HistoryFieldType::COEFFICIENT);
+  /// DESCRIPTION: viscous Drag coefficient
+  AddHistoryOutput("DRAG_VISC",       "CD_visc",   ScreenOutputFormat::FIXED, "AERO_COEFF", "Viscous drag coefficient on all surfaces set with MARKER_MONITORING", HistoryFieldType::COEFFICIENT);
```

Comment:
Can we keep this one for another day? It's not consistent to have viscous contributions only for one coefficient.

Reply:
Yeah sure, I added it here just because HLPW5 requires the history of the viscous drag coefficient. We could leave it here for those who want to use this branch for HLPW5 computations and then remove it just before the merge. You tell me.

Reply:
Yep we can do it that way. If you output the forces breakdown file you should have the different contributions for each coefficient.

Reply:
Yeah but I do not have it in the history file, right?

Reply:
Not at the moment